rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -73,3 +74,55 @@ def test_generate_chunks():
         ],
     ]
     assert generated_chunks == expected_chunks
+
+
+def test_last_queried_ts(tokens):
+    """
+    Checks that after detecting evm tokens last_queried_timestamp is updated and there
+    are no duplicates.
+    """
+    # We don't need to query chain here, so mock tokens list
+    evm_tokens_patch = patch(
+        'rotkehlchen.globaldb.handler.GlobalDBHandler.get_evm_tokens',
+        new=lambda _, chain_id, exceptions=None, except_protocols=None, protocol=None: [],
+    )
+    beginning = ts_now()
+    address = '0x4bBa290826C253BD854121346c370a9886d1bC26'
+    with evm_tokens_patch:
+        # Detect for the first time
+        tokens.detect_tokens(
+            only_cache=False,
+            addresses=[address],
+        )
+        after_first_query = tokens.db.conn.execute(
+            'SELECT key, value FROM evm_accounts_details',
+        ).fetchall()
+        assert len(after_first_query) == 1
+        assert after_first_query[0][0] == 'last_queried_timestamp'
+        assert int(after_first_query[0][1]) >= beginning
+
+        # Since no remote queries happen, detect_tokens executes instantly. Timestamp is saved
+        # in seconds and thus we have modify it. Otherwise the second call to detect_tokens
+        # would use the same timestamp.
+        tokens.db.conn.execute(
+            'UPDATE evm_accounts_details SET value=? WHERE '
+            'account=? AND chain_id=1 AND key=?',
+            (123456, address, 'last_queried_timestamp'),
+        )
+        # Make sure that it was modified
+        after_updating = tokens.db.conn.execute(
+            'SELECT key, value FROM evm_accounts_details',
+        ).fetchall()
+        assert int(after_updating[0][1]) == 123456
```

Comment:
Don't do this. This is really bad test code. Just move the time with pytest freezegun, a module we already use.
```suggestion
freezer.move_to('Some time in the future')
```